Clarify the rationale and purpose of touch-action

patrickhlauke opened the pull request:
- moved the note about viewport manipulation from the "Declaring candidate regions for default direct manipulation behaviors" section directly to the "Attributes and Default Actions" section, and made the text normative rather than informative (by not having it as a note, but actual part of the normative prose)
- expanded the sentence to more clearly convey what this means - by not having viewport manipulations treated/counted as a default action for pointer events, panning/zooming can't be prevented by cancelling a pointer event, and that instead authors must use `touch-action`
- tweaked the intro section to "Declaring candidate regions for default direct manipulation behaviors" to reiterate this concept/rationale
- removed one confusing instance of "default behavior" and changed to just "behavior" in a note in the touch-action section

Closes https://github.com/w3c/pointerevents/issues/366, closes https://github.com/w3c/pointerevents/issues/368


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/376.html" title="Last updated on May 19, 2021, 12:18 AM UTC (b1f888f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/376/284ec3f...b1f888f.html" title="Last updated on May 19, 2021, 12:18 AM UTC (b1f888f)">Diff</a>

patrickhlauke committed:
Clarify the rationale and purpose of touch-action

- moved the note about viewport manipulation from the "Declaring candidate regions for default direct manipulation behaviors" section directly to the "Attributes and Default Actions" section, and made the text normative rather than informative (by not having it as a note, but actual part of the normative prose)
- expanded the sentence to more clearly convey what this means - by not having viewport manipulations treated/counted as a default action for pointer events, panning/zooming can't be prevented by cancelling a pointer event, and that instead authors must use `touch-action`
- tweaked the intro section to "Declaring candidate regions for default direct manipulation behaviors" to reiterate this concept/rationale
- removed one confusing instance of "default behavior" and changed to just "behavior" in a note in the touch-action section

Closes https://github.com/w3c/pointerevents/issues/366 https://github.com/w3c/pointerevents/issues/368

patrickhlauke committed:
Already add this PR to revision history

... saves time for when it gets (hopefully) merged. Needs to be listed as it is substantive (turns the informative note into a normative requirement)

patrickhlauke committed:
Merge branch 'gh-pages' into patrickhlauke-issue368

mustaqahmed commented on index.html:
```diff
@@ -756,8 +758,7 @@ <h2>Extensions to the `Navigator` interface</h2>
     </section>
     <section>
         <h1>Declaring candidate regions for default direct manipulation behaviors</h1>
```
Perhaps remove the "default" in this section header too?

patrickhlauke replied:
good idea, done

mustaqahmed reviewed (approved):
This makes Sec8 well-connected with the rest of the spec at last.  Thanks for looking into this!

I checked all usage of "default...behavior".  I found two other spots where the word "default" could be removed: Sec1 para3 and (marked below) Sec8 header.  Please consider removing them if makes sense to you too.

patrickhlauke committed:
Removing two more instances of "default ... behaviour"

there are still a few cases of "default direct manipulation behaviour" present in prose, but I think in those contexts, they make sense to be used and don't confuse matters with "default action"
